### PR TITLE
facebook sharing

### PR DESCRIPTION
**Pull request by jtrudell**
First, it took me an insane amount of time to add share and like buttons, but it was a great learning experience. Second, one day I will have to sit down with you and just go through the whole MEAN stack, because I have read 80 tutorials at this point and I still don't know how to get the photos views into the server layout view.

### CODE REVIEW

File: frontend/public/modules/photos/views/view-photo.client.view.html [hunk 1]
```diff
@@ -1,3 +1,11 @@
 <section data-ng-controller="PhotosController" data-ng-init="findOne()">
+<<<<<<< HEAD
```
Comment (bmt) [inline frontend/public/modules/photos/views/view-photo.client.view.html]: You have a merge conflict that needs to be fixed here.

File: frontend/public/modules/photos/views/view-event.client.view.html [hunk 2]
```diff
@@ -1,3 +1,7 @@
-<section data-ng-controller="PhotosController" data-ng-init="find()">
-  TODO: Put photos here.
+ <section data-ng-controller="PhotosController" data-ng-init="find()">
```
Comment (bmt) [inline frontend/public/modules/photos/views/view-event.client.view.html]: Please remove the extra space before the <section>

File: frontend/app/views/index.server.view.html [hunk 3]
```diff
@@ -1,5 +1,12 @@
 {% extends 'layout.server.view.html' %}
 
 {% block content %}
-	<section data-ui-view></section>
+
+<section data-ui-view>
```
Comment (bmt) [inline frontend/app/views/index.server.view.html]: Lets leave this one off for now -- no one should be using this page directly.

File: frontend/app/views/layout.server.view.html [hunk 4]
```diff
@@ -31,10 +31,15 @@
...
 <body class="ng-cloak">
-	<header data-ng-include="'/modules/core/views/header.client.view.html'" class="navbar navbar-fixed-top"></header>
+
+  <header data-ng-include="'/modules/core/views/header.client.view.html'"
+  class="navbar navbar-fixed-top navbar-inverse"></header>
+
 	<section class="content">
 		<section class="container">
-			{% block content %}{% endblock %}
+			{% block content %}
```
Comment (bmt) [inline frontend/app/views/layout.server.view.html]: Is this necessary or were you just testing?

Comment (jtrudell) [inline frontend/app/views/layout.server.view.html]: Testing, should have deleted

Jen

jtrudell@gmail.com 
312-218-6381 (cell)
jtrudell78 (Skype)

File: frontend/public/modules/photos/views/view-photo.client.view.html (hunk 1 above)
Comment (jtrudell) [inline frontend/public/modules/photos/views/view-photo.client.view.html]: Got it. Will clean this up at lunch. 

Jen

jtrudell@gmail.com 
312-218-6381 (cell)
jtrudell78 (Skype)

File: frontend/app/views/layout.server.view.html [hunk 5]
```diff
@@ -32,6 +32,7 @@
 
 <body class="ng-cloak">
 	<header data-ng-include="'/modules/core/views/header.client.view.html'" class="navbar navbar-fixed-top"></header>
+
```
Comment (bmt) [inline frontend/app/views/layout.server.view.html]: Looks like you can remove this change completely now.

**Pull request merged**